Solutions/ESETPROTECT/Analytic Rules/ESETThreatDetected.yaml

```diff
@@ -0,0 +1,41 @@
+id: 64badfab-1dd8-4491-927b-3ca206fa9a17
+name: Threats detected by ESET
+description: |
+  'Escalates threats detected by ESET.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ESETPROTECT
+    dataTypes:
+      - Syslog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - CredentialAccess
+  - PrivilegeEscalation
+query: |
```

Comment:
Relevant Technique field is missing, can you please update the same and [refer this link ](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide)for details.